save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ExecutionView.kt

```diff
@@ -290,25 +297,38 @@ class ExecutionView : AbstractView<ExecutionProps, ExecutionState>(false) {
                     th {
                         attrs.colSpan = "${tableInstance.columns.size}"
                         child(testExecutionFiltersRow(
-                            initialValue = state.status?.name ?: "ANY"
-                        ) { value ->
-                            if (value == "ANY") {
-                                setState {
-                                    status = null
+                            initialValueStatus = state.status?.name ?: "ANY",
```

Comment:
Please extrat this `ANY` into a `const val` in `FiltersRow.kt`, I forgot to do this when initially implementing it.

Reply:
Fixed